easybuild/easyconfigs/p/PLINK/PLINK-1.07-x86_64-goolf-1.7.20.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PackedBinary'
+
+name = 'PLINK'
+version = '1.07-x86_64'
+
+homepage = 'http://pngu.mgh.harvard.edu/~purcell/plink/'
+description = 'plink-1.07-src: Whole-genome association analysis toolset'
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(namelower)s-%(version)s.zip']
+source_urls = ['http://pngu.mgh.harvard.edu/~purcell/plink/dist/']
+
+checksums = ['fd0bafeda42151b20534e4f97b0d97df']
+
+files_to_copy = [
```

Comment:
Ah, in that case, keep it as is, it's better to have similar style across versions, indeed.